packages/build-info/src/frameworks/angular.ts

```diff
@@ -32,6 +32,15 @@ export class Angular extends BaseFramework implements Framework {
     if (this.detected) {
       if (this.version && gte(this.version, '17.0.0-rc')) {
         this.plugins.push('@netlify/angular-runtime')
+        const angularJson = await this.project.fs.gracefullyReadFile('angular.json')
+        if (angularJson) {
+          const { projects, defaultProject } = JSON.parse(angularJson)
+          const project = projects[defaultProject ?? Object.keys(projects)[0]]
```

Comment:
I guess this might go slightly wrong if there's more than one project?

Reply:
Totally! Maybe we should add a warning in case there's multiple projects.